airbyte-server/src/main/java/io/airbyte/server/ConfigDumpImporter.java

```diff
@@ -119,7 +106,6 @@ public ImportRead importDataWithSeed(String targetVersion, File archive, Path se
   private ImportRead importDataInternal(String targetVersion, File archive, Optional<Path> seedPath) {
     Preconditions.checkNotNull(seedPath);
 
-    final Optional<UUID> previousCustomerIdOptional = getCurrentCustomerId();
```

Comment:
no longer trying to track and alias previous workspace. in the automigration case this was superfluous anyway. in the case of using the import endpoint this could mess up metrics a little. see the note about the trade off we are making in the main description of the pr.